docs/content/2.deploy/providers/alwaysdata.md

```diff
@@ -0,0 +1,36 @@
+# alwaysdata
+
+Deploy Nitro apps to alwaysdata.
+
+**Preset:** `alwaysdata` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+Nitro supports deploying on the [alwaysdata](https://www.alwaysdata.com/en/) platform with minimal configuration.
+
+## Set up application
+
+### Pre-requisites
+
+1. [Register a new profile](https://www.alwaysdata.com/en/register/) on alwaysdata platform if you don't have one.
+
+2. Get a free 100Mb plan to host your app.
+
+### Local deployment
+
+1. Build your project locally with `NITRO_PRESET=alwaysdata yarn build`
+
+2. [Upload your app](https://help.alwaysdata.com/en/remote-access/) to your account in its own directory (e.g. `$HOME/www/my-app`).
+
+3. On your admin panel, [create a new site](https://admin.alwaysdata.com/site/add/) for your app with the following features:
+   - *Addresses*: `[account-name].alwaysdata.net`
+   - *Type*: Node.js
+   - *Command*: `node ./output/server/index.mjs`
```

Comment:
There is no instructions how to upload website but it is `.output` by default i expect directories inside it (public/server) being copied to the hosting

Reply:
Correct, I added a mention in the 2. above to add clarity on how deploying, for example using SSH